homeworks/14/instagram-api/RegisterServlet.java

```diff
@@ -0,0 +1,57 @@
+package com.example.instagram;
+
+import jakarta.servlet.*;
+import jakarta.servlet.http.*;
+import jakarta.servlet.annotation.*;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.sql.*;
+
+
+@WebServlet(name = "register", value = "/register")
+public class RegisterServlet extends HttpServlet {
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        PrintWriter out = response.getWriter();

```

Comment:
It is good practice to close resources such as PrintWriters when you are finished using them.